### DISCUSSION TITLE
cannot access minio container

### DISCUSSION THREAD

**Discussion by widojansen** [Budibase/budibase#4891] [Help]
Hi,

I am trying to access the minio service as described in the docs.
Set MINIO_BROWSER to "on". Restart all containers.

I see in the logs the dynamic console port, for example 40093.

When I try to access Minio with http:\\<IP>:10000\minio I receive a browser error that the website cannot be found.
The URL in the browser is minio-service:40093

What am I missing here?

**Comment by mjashanks** [comment]
Hey @widojansen .

I just tried this and it worked ok. I set the `MINIO_BROWSER: "on"` in the `docker-componse.yaml`, restarted containers, then accessed on `http://localhost:10000/minio`

**Comment by widojansen** [comment]
Hey @mjashanks 

Thanks for your test but I tested both on a clean VPS and on my Mac with a new project. 
But not working. Same error.

This is shown in the log of the bbproxy: 
bbproxy               | 90.187.26.117 - - [15/Mar/2022:13:07:08 +0000] "GET /minio HTTP/1.1" 307 62 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.3 Safari/605.1.15"
bbproxy               | 2022/03/15 13:07:34 [info] 33#33: *27 client 90.187.26.117 closed keepalive connection
bbproxy               | 2022/03/15 13:07:34 [info] 35#35: *28 client 90.187.26.117 closed keepalive connection
bbproxy               | 2022/03/15 13:07:34 [info] 33#33: *30 client 90.187.26.117 closed keepalive connection

Then I get the error cannot open page in the browser and the address shown is 
minio-service:some port

I can make it work when I modify the minio-service container in docker-compose.yaml:
1) Add --console-address ":43000" to the command
2) Add ports: - "43000:43000"

Then I can access the minio console of the container and I see the apps files.

**Comment by RASS00** [comment]
Same for me, this shows when I try to localhost//:10000/minio

![image](https://user-images.githubusercontent.com/82243352/158688676-a612272f-b5bd-4394-94d7-bc2736d310d4.png)

**Reply by widojansen** [reply]
In the minio container it seems these calls are redirected to the dynamic console port.

So i added a static console port and port settings to the docker-composers.yaml. See my previous message.

After that I can access the minio container.

Need to test more and maybe modify the proxy container.

**Reply by AndyGUK** [reply]
I seem to be getting a similar issue. 

Is using Caddy likely to need additional steps to those described in the Budibase docs: https://docs.budibase.com/docs/reverse-proxy

**Reply by widojansen** [reply]
The problem occurs also without using a reverse proxy like Caddy or Nginx.

So the issue is in the containers setup. Not sure on Minio and/or the BBProxy.

Like I described above I can bypass this problem by adjusting the Minio settings in docker-compose.yaml

Of course when I want to use a reverse proxy after the modifications I would need to change the reverse proxy setup.

Coming week I will run some tests and I will share my findings.
